Incompatible types: List<LinearGradientManager> cannot be converted to List<ViewManager>

Reactnative: 0.64
Java: 11
----------------------------
I am facing this issue:
node_modules\react-native-linear-gradient\android\src\main\java\com\BV\LinearGradient\LinearGradientPackage.java:23: error: incompatible types: List<LinearGradientManager> cannot be converted to List<ViewManager>
        return Collections.singletonList(new LinearGradientManager());

Just changed it as below works fine, why????:
node_modules/react-native-linear-gradient/android/src/main/java/com/BV/LinearGradient/LinearGradientPackage.java
@NonNull
@Override
public List<ViewManager> createViewManagers(@NonNull ReactApplicationContext reactContext) {
    // return Collections.singletonList(new LinearGradientManager()); //commment this line
    return Collections.<ViewManager>singletonList(new LinearGradientManager()); //add this line
}

I don't know everyone is facing this problem or not?

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -39,6 +39,10 @@ android {
     lintOptions {
         abortOnError false
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
+    }
 }
 
 repositories {

--- a/android/src/main/java/com/BV/LinearGradient/LinearGradientPackage.java
+++ b/android/src/main/java/com/BV/LinearGradient/LinearGradientPackage.java
@@ -20,6 +20,6 @@ public class LinearGradientPackage implements ReactPackage {
     @NonNull
     @Override
     public List<ViewManager> createViewManagers(@NonNull ReactApplicationContext reactContext) {
-        return Collections.singletonList(new LinearGradientManager());
+        return Collections.<ViewManager>singletonList(new LinearGradientManager());
     }
 }

